### PR TITLE
Bug 1757709: CVE-2019-11253 billion laughs

### PR DESCRIPTION
**Pull request by jottofar**
Process followed:
1. modified and committed glide.yaml
2. 'make update-deps' 
   - modified glide.yaml per 3.11 to contain https://github.com/openshift/gssapi.git version release-2.6.3
  - commited
3. cherry-picked b067091 "UPSTREAM: 83261...", fixed build errors, committed
4. cherry=picked b1037fc "uni test changes", committed

### CODE REVIEW

File: pkg/apps/strategy/support/acceptor.go [hunk 1]
```diff
@@ -1,84 +0,0 @@
...
-	corev1 "k8s.io/api/core/v1"
-	kerrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/apimachinery/pkg/watch"
-	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
-)
-
-// NewAcceptAvailablePods makes a new acceptAvailablePods from a real client.
-func NewAcceptAvailablePods(
```
Comment (deads2k) [inline pkg/apps/strategy/support/acceptor.go]: why did this go away?

Comment (jottofar) [reply pkg/apps/strategy/support/acceptor.go]: code is in pkg/apps/strategy/support/lifecycle.go

File: pkg/apps/strategy/support/acceptor.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
+package support
```
Comment (deads2k) [inline pkg/apps/strategy/support/acceptor.go]: looks like you added back exactly the same thing here

**Comment by deads2k** [comment]
questioned answered via slack

/lgtm

**Pull request merged**